modules/form/src/components/adapters.tsx

```diff
@@ -128,6 +128,27 @@ export const TextFieldAdapter = (props:FieldRenderProps<any> ): ReactElement =>
     );
 };
 
+export const ScopeFieldAdapter = (props:FieldRenderProps<any> ): ReactElement => {
+
+    const { childFieldProps, input } = props;
+
+    return (
+        <>
+            <Scopes
+                defaultValue={ childFieldProps.defaultValue }
```

Comment:
Lets add a null check.`childFieldProps?.defaultValue` 
Check in other places as well